ui/src/app/pages/analysis/runtime-panel.tsx

```diff
@@ -1255,7 +1255,19 @@ const DataProcConfigSelector = ({
         </FlexRow>
         <FlexRow style={styles.labelAndInput}>
           <label style={styles.label} htmlFor='num-preemptible'>
-            Preemptible
+            Preemptible workers
+            <TooltipTrigger
+              content={
+                <div>
+                  Preemptible secondary workers can be added in addition to
+                  primary workers. They are less expensive than primary workers
+                  but may be removed from the cluster if they are required by
+                  Google Cloud for other tasks. This may affect job stability
+                </div>
+              }
+            >
+              <InfoIcon style={{ marginLeft: '0.1rem' }} />
```

Comment:
what do you think about changing the size (height and width) of infoicon to say 18px?

Reply:
So we can change the height and width in the style app, <InfoIcon style={{ marginLeft: '0.1rem', height: '18px',
                width: '18px',}} />

the value passed in style object will take preference over the height and weight defined in InfoIcon
Doing the above i could see the label as 
<img width="491" alt="Screen Shot 2022-09-13 at 1 24 10 PM" src="https://user-images.githubusercontent.com/34481816/189968616-43583138-d234-4222-b40a-54020cd09943.png">

Reply:
You're right, I had misread the component!

Interestingly, when I use that same styling it appears as an oval for me, which is why I thought there was an issue before. I think this is an issue with my environment though (I'm accessing a Linux machine through my computer and have been running into rendering issues) so I'll make this change.

Reply:
Done!